locksmith/src/graphql/datasource/keysByQuery.ts

```diff
@@ -8,14 +8,13 @@ export type KeyFilter = 'all' | 'active' | 'expired' | 'keyId'
 const keyholdersByKeyIdQuery = gql`
   query Lock(
     $addresses: [String!]
-    $expireTimestamp: BigInt! = 0
     $first: Int! = 100
     $skip: Int! = 0
     $keyId: BigInt
   ) {
     locks(where: { address_in: $addresses }) {
       keys(
-        where: { expiration_gt: $expireTimestamp, keyId: $keyId }
+        where: { expiration_gt: 0, keyId: $keyId }
```

Comment:
though, I'm wondering why the default value isn't working here.

Reply:
because tha value is always passed
1: https://github.com/unlock-protocol/unlock/blob/8b18db8ed8e3ba03ba4a737eecf333b774c8eb25/locksmith/src/graphql/datasource/keysByQuery.ts#L156
2: https://github.com/unlock-protocol/unlock/blob/8b18db8ed8e3ba03ba4a737eecf333b774c8eb25/locksmith/src/graphql/datasource/keysByQuery.ts#L174

Reply:
You can just stop passing the value too in the code, right? I would remove that part because it's confusing to pass variables when they aren't respected in the query. 

Also, Are you sure this doesn't actually break any behaviour?

Reply:
need to pass the value because we have different query where the value is needed, best thing here I think to remove it in the specific query

Reply:
let meet update it in a better way